File: bubblesort.py
n=int(input('Enter the number of elements in the array : '))
a=[]
for i in range(n):
    val = input(f'Enter value for position {i}: ')
    a.append(val)
for i in range(len(a)):
    for j in range(len(a)-i-1):
        if (a[j] > a[j + 1]):
            a[j], a[j + 1] = a[j + 1], a[j]
print('The sorted array is : ',a)

File: bubblesort.c
#include <stdio.h>
int main()
{
    int a[10], i, j, temp;
    printf("Enter the elements in the array \n");
    for(i = 0; i < 10; i++)
    {
        printf("Enter value for index %d : ", i);
        scanf("%d", &a[i]);
    }
    for(i = 0; i < 10; i++)
    {
        for(j = 0; j < 10 - i - 1; j++)
        {
            if(a[j] > a[j + 1])
            {
                temp = a[j];
                a[j] = a[j + 1];
                a[j + 1] = temp;
            }
        }
    }

    printf("The sorted elements are: ");
    for(i = 0; i < 10; i++)
    {
        printf("%d\t", a[i]);
    }
    return 0;
}
